services/app-web/src/hooks/useContractForm.ts

```diff
@@ -0,0 +1,192 @@
+import { useState, useEffect} from 'react'
+import { usePage } from "../contexts/PageContext"
+import {
+    CreateContractInput,
+    useFetchContractQuery,
+    useCreateContractMutation,
+    useUpdateContractDraftRevisionMutation,
+    ContractDraftRevisionFormDataInput,
+    UpdateInformation,
+    Contract
+} from '../gen/gqlClient'
+import { recordJSException } from '../otelHelpers'
+import { handleApolloError } from '../gqlHelpers/apolloErrors'
+import { ApolloError } from '@apollo/client'
+import type { InterimState } from '../pages/StateSubmission/ErrorOrLoadingPage'
+
+
+type  UseContractForm = {
+    draftSubmission?: Contract
+    unlockInfo?: UpdateInformation
```

Comment:
Is this needed when we also returning to draft Submission? I'm a bit confused why this is surfaced as well